vnda/loaders/productList.ts

```diff
@@ -19,6 +19,9 @@ export interface Props {
   /** @description search for products that have certain tag */
   tags?: string[];
 
+  /** @description search for products that have certain type_tag */
+  typeTags?: { key?: string; value?: string }[] | undefined;
```

Comment:
```suggestion
  typeTags?: { key?: string; value?: string }[];
```

nit: I think the `typeTags?` already accounts for `| undefined`

Reply:
done :)